### ISSUE TITLE
Update `.hyper.js` file

### ISSUE THREAD

**Issue by jaebradley** [jaebradley/dotfiles#37]
```javascript
module.exports = {
  config: {
    opacity: 0.9,
    updateChannel: "canary",
    fontSize: 14,
    fontFamily: '"Hack Nerd Font", "Operator Mono", Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',
    fontWeight: 300,
    fontWeightBold: 700,
    cursorColor: "rgba(248,28,229,0.8)",
    cursorAccentColor: "#000",
    cursorShape: "BLOCK",
    cursorBlink: false,
    foregroundColor: "#fff",
    backgroundColor: "#000",
    selectionColor: "rgba(248,28,229,0.3)",
    borderColor: "#333",
    css: "",
    termCSS: "",
    showHamburgerMenu: "",
    showWindowControls: "left",
    padding: "12px 14px",
    colors: {
      black: "#000000",
      red: "#C51E14",
      green: "#1DC121",
      yellow: "#C7C329",
      blue: "#0A2FC4",
      magenta: "#C839C5",
      cyan: "#20C5C6",
      white: "#C7C7C7",
      lightBlack: "#686868",
      lightRed: "#FD6F6B",
      lightGreen: "#67F86F",
      lightYellow: "#FFFA72",
      lightBlue: "#6A76FB",
      lightMagenta: "#FD7CFC",
      lightCyan: "#68FDFE",
      lightWhite: "#FFFFFF"
    },
    shell: "",
    shellArgs: ["--login"],
    env: {},
    bell: "SOUND",
    copyOnSelect: false,
    defaultSSHApp: true
  },
  plugins: ["hyper-material-theme", "hyper-opacity"],
  localPlugins: [],
  keymaps: {}
};
```

**Comment by jaebradley** [comment]
A lot of the formatting from https://medium.com/@riceboyler/my-hyper-ohmyzsh-config-3e982db30210

**Comment by jaebradley** [comment]
Ensure 

```js
modifierKeys: {
  altIsMeta: true,
  altIsEscape: true,
}
```

**Comment by jaebradley** [comment]
Closed by #50
